knowledge_engine/knowledge_engine/storage/milvus_backend.py

```
@@ -728,6 +770,106 @@ def list_documents(
+                        "title": node.metadata.get("source_file", ""),
+                        "metadata_json": json.dumps(node.metadata),
+                    }
+                    for node in parent_nodes
+                ],
+            )
+            return {"stored_count": len(parent_nodes)}
+        finally:
+            try:
+                client.close()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)